proto/streamlit/proto/Arrow.proto

```diff
@@ -17,14 +17,31 @@
 syntax = "proto3";
 
 message Arrow {
-  bytes data = 1;
-  Styler styler = 2;
-  // width in CSS points
-  uint32 width = 3;
-  // height in CSS points
-  uint32 height = 4;
+  // The id of the widget, this is required if the dataframe is editable
+  string id = 1;
+  // The serialized arrow dataframe
+  bytes data = 2;
+  // Pandas styler information
+  Styler styler = 3;
+  // Width in CSS points
+  uint32 width = 4;
+  // Height in CSS points
+  uint32 height = 5;
   // If True, will overwrite the dataframe width to fit to container.
-  bool use_container_width = 5;
+  bool use_container_width = 6;
+  // Colum configuration as JSON
+  string columns = 7;
+  // Activate table editing
+  enum EditingMode {
```

Comment:
nit: this should probably be moved towards the bottom. 

Ref: https://github.com/streamlit/streamlit/blob/develop/proto/streamlit/proto/Alert.proto